code/game/objects/items/miscellaneous.dm

```diff
@@ -180,13 +180,30 @@
 	var/itemheld = FALSE
 	var/capacity = 2
 	var/maximum_size = 2 //one human, two pets, unlimited tiny mobs, but no big boys like megafauna
+	var/kidnappingcoefficient = 1
 
+/obj/item/clothing/head/that/bluespace/attackby(obj/item/W, mob/user, params)
+	. = ..()
+	if(istype(W, /obj/item/upgradewand))
```

Comment:
You should add a check to see if it is already upgraded before people buy 2 wands and try to double upgrade things.